src/pages/Transactions/transactionUtils.tsx

```diff
@@ -50,6 +50,19 @@ const isUserRejectionError = (error: any) => {
   return false;
 };
 
+const isLedgerError = (error: any) => {
+  if (error.code === -32603) {
+    // Error message when rejecting from ledger or blind signing and debug not enabled
+    if (
+      error.message.includes('Ledger: Unknown error while signing transaction')
+    ) {
+      return true;
+    }
+  }
+
+  return false;
+};
```

Comment:
@valefar-on-discord Oops sorry, was on mobile and missed the context...

So `-32603` is the error code... is there a reason we need to also check for the "Ledger:" string below? If so, I may consider just checking for `.toLowerCase().includes("ledger")` so any error message with "Ledger" in it will be flagged.

And also, we could simplify this to just return `error.code === -32603 && error.message.toLowerCase().includes("ledger")` as a boolean:

```suggestion
// Error message when rejecting from ledger or blind signing and debug not enabled
const isLedgerError = (error: any) =>
  error.code === -32603 && error.message.toLowerCase().includes("ledger")  {
```

Reply:
Yup yup, sounds good.

Thanks for taking a look!